app/client/src/routes/home.tsx

```diff
@@ -179,8 +178,14 @@ export function QueryBuilder() {
     <>
       {downloadConfirmationVisible && (
         <DownloadModal
+          dataId="attains"
+          disabled={
+            !profiles[profile].columns.has('reportingCycle') &&
+            Object.keys(queryParams.filters).length === 0
+          }
```

Comment:
```suggestion
          disabled={
            !profiles[profile].columns.has('reportingCycle') &&
            Object.keys(queryParams.filters).length === 0 &&
            profile !== 'tmdl'
          }
```

I'm a little torn on this one. The TMDL profile only has 12108 rows in it, so it seems a little silly to send them to the national downloads page, but at the same time this does offload some work from the EQ server.  

I'm ok with skipping this and discussing it more tomorrow.

Reply:
We could just forego this initial check, and just go by result count, since it would only apply to one case in the Actions profile.

Reply:
That sounds good to me.

Reply:
We could also duplicate the maximum query config to the server, and add that as a limit to the query. Then maybe change the client-side check to >=.

Reply:
I think I'm good with the way it is. We can discuss it more during today's meeting though.